src/test/data/entity/claimData.ts

```diff
@@ -61,3 +61,218 @@ export const claimData = {
     rows: []
   }
 }
+
+export const sampleClaimData = {
```

Comment:
do we need another sample? is it not possible to use existing objects and add what you need?